src/Controller/ModulesController.php

```diff
@@ -350,6 +350,26 @@ protected function saveRelated(array $relatedData, string $id): void
             $method = (string)Hash::get($rel, 'method');
             $relation = (string)Hash::get($rel, 'relation');
             $relatedIds = (array)Hash::get($rel, 'relatedIds');
+
+            foreach ($relatedIds as $key => &$data) {
+                if (!empty($data['attributes'])) {
+                    // avoid empty objects
+                    $saveObject = false;
+                    foreach ($data['attributes'] as $key => $field) {
+                        if ($key != 'status' && !empty($field)) {
+                            $saveObject = true;
+                        }
+                    }
+
+                    if ($saveObject == true) {
```

Comment:
minor, suggested by scrutinizer

Reply:
What exactly? 

You are saying to just use 
`if ($saveObject )` ?